modules/openapi-generator/src/main/resources/dart-dio/api.mustache

```diff
@@ -43,15 +43,15 @@ class {{classname}} {
         {{#queryParams}}
         queryParams[r'{{baseName}}'] = {{paramName}};
         {{/queryParams}}
-        queryParams.removeWhere((key, value) => value == null);
-        headerParams.removeWhere((key, value) => value == null);
+        queryParams.removeWhere((key, dynamic value) => value == null);
```

Comment:
if you put `dynamic` before `value` then for consistency we should have `String` before `key`

Reply:
Due to `implicit-dynamic: false`, you always have to be explicit about `dynamic`. But the `String` key is inferred.